### PR TITLE
update user documentation for using Helm chart

### LINKED ISSUE #406: Documentation for Kubeapps Helm Chart
Documentation we need to update:
- Installation/getting started guides
- Developer docs for new references (e.g. telepresence deployment names) (https://github.com/kubeapps/kubeapps/issues/438)

### PR DESCRIPTION
**Pull request by prydonius**
WIP: merge after v1.0.0-alpha.5 release

Will update developer docs in a separate PR

fixes #406

### CODE REVIEW

File: chart/kubeapps/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# <img src="https://github.com/kubeapps/kubeapps/raw/master/docs/img/logo.png" width="40" align="left"> Kubeapps
+
+[![Build Status](https://travis-ci.org/kubeapps/kubeapps.svg?branch=master)](https://travis-ci.org/kubeapps/kubeapps)
+
+[Kubeapps](https://kubeapps.com) is a web-based UI for deploying and managing
```
Comment (andresmgot) [inline chart/kubeapps/README.md]: why are you splitting the lines manually, is that a requirement for chart docs?

Comment (prydonius) [reply chart/kubeapps/README.md]: No specific reason, I reverted to be more consistent

File: README.md [hunk 2]
```diff
@@ -13,21 +13,24 @@ Kubeapps is a web-based UI for deploying and managing applications in Kubernetes
...
+Kubeapps assumes a working Kubernetes cluster (v1.8+) and [`Helm`](https://helm.sh/) (2.9.1+) installed in your cluster. Kubeapps has been tested with Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), `minikube` and Docker for Desktop Kubernetes. Kubeapps works on RBAC-enabled clusters and this configuration is encouraged for a more secure install.
 
 > On GKE, you must either be an "Owner" or have the "Container Engine Admin" role in order to install Kubeapps.
 
-The simplest way to try Kubeapps is to deploy it with the Kubeapps Installer on [minikube](https://github.com/kubernetes/minikube). Assuming you are using Linux or OS X, run the following commands to download and install the Kubeapps Installer binary:
+Use the Helm chart to install the latest version of Kubeapps:
```
Comment (migmartri) [inline README.md]: Maybe add a link to the chart readme by saying something like "Click here for more information on how to configure your deployment"

Comment (migmartri) [reply README.md]: actually NM, I saw that the link is down there. I am wondering if we should move it up here.

> For information on installing and configuring Kubeapps, checkout the [chart README](chart/kubeapps/README.md).

Comment (prydonius) [reply README.md]: agreed, I moved it up

**Pull request merged**